locale/en.json

```diff
@@ -997,7 +997,9 @@
   "BOTH_VULNERABLE_MESSAGE": "Deleting this encounter would result in assigned individuals and/or parent sightings being deleted. Are you sure you want to continue?",
   "DETACH_ENCOUNTER_FROM_INDIVIDUAL": "Detach this sighting from this individual",
   "DETACH_ENCOUNTER_FROM_INDIVIDUAL_DISABLED": "Cannot detach the final sighting from this individual",
+  "DUPLICATE_RELATIONSHIPS_FOUND_AND_REMOVED": "Duplicate relationships in one or more categories identified and removed",
+  "UNNAMED_INDIVIDUAL": "Unnamed individual",
+  "FEATURED_ASSET": "Featured asset",
```

Comment:
let's use "Top photograph" for user-facing stuff

Reply:
Done